dali/python/nvidia/dali/plugin/pytorch.py

```diff
@@ -413,6 +413,287 @@ def __init__(self,
                                                          last_batch_policy=last_batch_policy,
                                                          prepare_first_batch=prepare_first_batch)
 
+###############################################
+###############################################
+# Gluon API
+###############################################
+###############################################
+
+
+class DALIGluonIterator(_DaliBaseIterator):
```

Comment:
[Gluon](https://mxnet.apache.org/versions/1.6/api/python/docs/api/gluon/index.html) is an MXNet framework specific module. In this case I would call it DALIRadgedIterator or just extend the existing ones for PyTorch.

Reply:
Typo. My bad.

Reply:
Sure, we were not aware that the `Gluon` is MXNet-specific, thanks for pointing that out. `DaliRaggedIterator` sounds like a good name, we'll rename it so.